### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by hari-kuriakose**
Added GitHub standard issue templates for bug report and feature request.

This is NOT same as the PR template.
This should be enough for now to pass check for issue templates in [community standards](https://github.com/Zipstack/unstract/community).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+A clear and concise description of what the bug is.
+
+## To reproduce
+Steps to reproduce the behavior.
+
+## Expected behavior
+A clear and concise description of what you expected to happen.
+
+## Environment details
+ - Version: [e.g. 0.41.2]
```
Comment (chandrasekharan-zipstack) [inline .github/ISSUE_TEMPLATE/bug_report.md:20]: We need to display the app version more clearly to the user - preferably at both the FE and BE while running the images

**Review by chandrasekharan-zipstack — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (ritwik-g) [reply .github/ISSUE_TEMPLATE/bug_report.md:20]: @hari-kuriakose @chandrasekharan-zipstack we could consider adding this in to the run script as a start.

**Review by ritwik-g — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (hari-kuriakose) [reply .github/ISSUE_TEMPLATE/bug_report.md:20]: @chandrasekharan-zipstack @ritwik-g Couldn't agree more.

**Pull request merged**